.github/workflows/plugin.yaml

```diff
@@ -20,6 +24,12 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Set profile
         run: export PLZ_CONFIG_PROFILE=ci && echo $PATH
+      - name: Build and use in-repo pex tool
+        if: matrix.pex-tool == 'in-repo'
+        run: |
+          ./pleasew build //tools/please_pex
+          cp $(./pleasew query outputs //tools/please_pex) $HOME/please_pex
```

Comment:
We can't use `.plzconfig.tool_dev` here because the repo directory doesn't persist across steps - the runner's home directory does, though, so we can copy `please_pex` there.